akka-actor-testkit-typed/src/test/scala/docs/akka/actor/testkit/typed/scaladsl/AsyncTestingExampleSpec.scala

```diff
@@ -24,10 +32,27 @@ object AsyncTestingExampleSpec {
     }
   }
   //#under-test
+
+  //#under-test-2
+  case class Message(i: Int, replyTo: ActorRef[Try[Int]])
+
+  class Producer(publisher: ActorRef[Message])(implicit scheduler: Scheduler) {
+
+    def produce(messages: Int)(implicit timeout: Timeout): Unit = {
+      (0 until messages) foreach publish
```

Comment:
I think we try to avoid this style unless it is a symbolic operator (`!`, `+`), so `.foreach(publish)` would be nicer on the Scala beginners eyes.